stores/firmwareStore.ts

```diff
@@ -32,7 +32,7 @@ export const useFirmwareStore = defineStore('firmware', {
             pullRequests: new Array<FirmwareResource>(),
             selectedFirmware: <FirmwareResource | undefined>{},
             selectedFile: <File | undefined>{},
-            baudRate: 115200,
+            baudRate: 460800,
```

Comment:
Default should remain 115200 for the fact of the post flash serial monitoring having that rate.

Reply:
That's fair.  Is the post flash serial monitoring useful (or confusing to new users)?  I always change my baud rate so it just prints nonsense.